app/src/main/java/com/github/mik629/android/fundamentals/ui/utils/Utils.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.mik629.android.fundamentals.ui.utils
+
+import android.content.Context
+import androidx.annotation.ColorInt
+import androidx.core.content.ContextCompat
+import com.github.mik629.android.fundamentals.R
+import com.github.mik629.android.fundamentals.databinding.RatingBarBinding
+
+@Suppress("NOTHING_TO_INLINE")
+inline fun RatingBarBinding.setRating(context: Context, value: Float) {
+    @ColorInt val grey = ContextCompat.getColor(context, R.color.grey)
+    arrayOf(starOne, starTwo, starThree, starFour, starFive)
```

Comment:
u missed my comment related to recycler example?

Reply:
?

Reply:
Do I need to change smth here or this had fixed it:         
```
arrayOf(starOne, starTwo, starThree, starFour, starFive)
        .forEachIndexed { index, star ->
            if (value < index) {
                star.setColorFilter(grey)
            } else {
                star.setColorFilter(pink)
            }
        }
```